applications/FluidDynamicsApplication/symbolic_generation/two_fluid_navier_stokes/two_fluid_navier_stokes.cpp

```diff
@@ -0,0 +1,2303 @@
+//    |  /           |
```

Comment:
This file should be removed. In the symbolic generation folder it is enough to have the template cpp as this one is identical to the source file in the `custom_elements` folder.